Tests/NumLib/TestComponentNorms.cpp

```diff
@@ -143,7 +143,7 @@ TEST(MPITest_NumLib, ComponentNormMultiComponent1)
     unsigned const num_components = 3;
     auto const norm_type = MathLib::VecNormType::NORM1;
     auto const tolerance =
-        num_components * 600 * std::numeric_limits<double>::epsilon();
```

Comment:
That sounds sound :smile: (Now the reference solution (computed with Eigen's routines) might use 80 bit at intermediate stages, whereas the component-wise computation only uses the 64 bit double precision everywhere.)